lz4-compress/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "lz4-compress-targets"
+version = "0.0.0"
+publish = false
+
+[dependencies]
+libfuzzer-sys = { git = "https://github.com/rust-fuzz/libfuzzer-sys.git" }
+lz4-compress = "0.1.0"
```

Comment:
Would you mind changing this to the git repository?

See https://github.com/rust-fuzz/targets/pull/18